package.json

```diff
@@ -34,33 +34,12 @@
     "@babel/preset-env": "7.3.4",
     "@babel/preset-react": "7.0.0",
     "@babel/runtime": "7.3.4",
-    "@wordpress/a11y": "2.0.2",
-    "@wordpress/api-fetch": "2.2.8",
     "@wordpress/babel-plugin-import-jsx-pragma": "1.1.3",
-    "@wordpress/blob": "2.1.0",
     "@wordpress/block-library": "2.2.16",
     "@wordpress/blocks": "6.0.7",
     "@wordpress/components": "7.0.8",
-    "@wordpress/compose": "3.0.1",
-    "@wordpress/core-data": "2.0.17",
-    "@wordpress/data": "4.2.1",
     "@wordpress/date": "3.0.1",
-    "@wordpress/deprecated": "2.0.5",
-    "@wordpress/dom": "2.0.8",
     "@wordpress/edit-post": "3.1.11",
-    "@wordpress/editor": "9.0.11",
-    "@wordpress/element": "2.1.9",
-    "@wordpress/format-library": "1.2.14",
-    "@wordpress/hooks": "2.0.5",
-    "@wordpress/i18n": "3.1.1",
-    "@wordpress/keycodes": "2.0.6",
-    "@wordpress/notices": "1.1.3",
-    "@wordpress/nux": "3.0.9",
-    "@wordpress/plugins": "2.0.11",
-    "@wordpress/rich-text": "3.0.7",
-    "@wordpress/token-list": "1.1.0",
-    "@wordpress/url": "2.3.3",
-    "@wordpress/viewport": "2.1.1",
```

Comment:
Why are we changing dependencies in this PR? How did we arrive at this list?

These deps are likely extraneous after #31199 and the removal of Gutenberg-in-Calypso, but the don't seem related to the changes here.

Reply:
Fair point, it probably makes sense to tackle this in a different PR.

Reply:
Nice! Good call.